Update site-configuration-settings.rst (#7656)
mattermost/docs: source/configure/site-configuration-settings.rst

  :start-after: :nosearch:

+---------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+

> | - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: type it here **Site Configuration > Users and Teams**              |

Reviewer: ```suggestion
| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |
```

+---------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+

> | - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |

Reviewer: ```suggestion
| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: type it here **Site Configuration > Users and Teams**              |
```

> |   and users can @mention custom user groups in Mattermost conversations.                          | - ``config.json`` setting: N/A                                              |

Reviewer: ```suggestion
|   and users can @mention custom user groups in Mattermost conversations.                          | - ``config.json`` setting: actual setting name                                              |
```
